web/custom-plugins/forecast-of-response/src/components/Result.tsx

```diff
@@ -212,16 +257,18 @@ export default (props: {
     setGraphInfo([])
   }, [result])
 
-  function removeGraph(name: string) {
+  function removeGraph(name: string, uuid?: string) {
```

Comment:
Looks too me that graphs always has uuid. So this function could be simplified

Reply:
Sort of. Will have to refactor further down (`removeGraph`) if we want to achieve that.
```
function removeGraph(name: string, uuid?: string) {
   [...]
    chartData.forEach((dataPoint: any) => {
      delete dataPoint[name]
      newDataDict[dataPoint.timestamp] = dataPoint
    })
```